src/api/schema.ts

```diff
@@ -522,9 +515,24 @@ export const lists = {
         ref: 'Application',
         many: true,
       }),
+      accountStatus: relationship({ ref: 'AccountStatus.account' }),
     },
     access: allowAll,
   }),
+  AccountStatus: {
+    fields: {
+      account: relationship({ ref: 'Account.accountStatus' }),
+      isVerified: checkbox({ defaultValue: false }),
+      /**
+       * isBlocked flag will only be true if the account has:
+       * - repeatedly attempted sign in
+       * - repeatedly attempted password reset request
+       */
+      isBlocked: checkbox({ defaultValue: false }),
+      isInactivated: checkbox({ defaultValue: false }),
```

Comment:
Can this be structured to be dynamic instead of individual status columns? If we have e.g 6-10 statuses, we'd have to refactor.

Could be just like `TotalContractValue` or `CoverPeriod`,

Reply:
Had a team chat about this - the approach we have here feels the best, simplest approach.

If there are more statuses added this could become troublesome and should probably be split up more into 2x different tables related to an account. However, it is extremely unlikey for us to require any additional statuses.

At this stage, other approaches feel like over engineering.

If we need to have a different, more split approach, this shouldn't be to cumbersome to achieve in the future.